.github/workflows/build.yml

```diff
@@ -0,0 +1,42 @@
+name: build
+on:
+  # Enable manually triggering this workflow via the API or web UI
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+    tags:
+      - v*
```

Comment:
Is it really required? I think we can skip it if we already run it on main commits.

Reply:
I don't know. This is inherited from the history of the repo, I just moved stuff around and bumped some stuff. Happy to remove it indeed 👍🏻

Reply:
It is not required (old process). Please, remove it. Thank you!

Reply:
Yeah, let's keep it simple.